### PR TITLE
Search books

### PR DESCRIPTION
**Pull request by gurralaharika21**
@rehana-ms @LaxmiNarayanaMurthyVemuri @Manideepnetha @dharshakch97-cloud 

please review my branch search page of project1.

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -68,8 +70,51 @@ def auth():
...
+def search():
+    text = ""
+    
+    if request.method == "POST":
+        word = request.form['searchbox']
+        choice = request.form['choice']
+        if len(word) > 0:
+            if choice == "isbn":
+                print(word)
+                query = db.query(Book).filter(Book.isbn.like(f'%{word}%'))
```
Comment (rehana-ms) [inline project1/application.py:87]: Can you display the search results for a search query: lion. pls share the screenshot.

Comment (gurralaharika21) [reply project1/application.py:87]: Ma'am is that in the title's o rin  author-name.

Comment (gurralaharika21) [reply project1/application.py:87]: Screenshot (480)
These are the reults of lion search in through title ma'am.

Comment (rehana-ms) [reply project1/application.py:87]: everything looks good except logout button can be on the top right corner instead of bottom.

**Comment by gurralaharika21** [comment]
yeah, ma'am, I have changed it to the top right corner.

**Comment by rehana-ms** [comment]
good

**Pull request merged**